src/app/features/tech-record/components/tech-record-title/tech-record-title.component.html

```diff
@@ -11,15 +11,16 @@ <h1 class="govuk-heading-l">
     <dd id="vehicleType" class="govuk-summary-list__value">
```

Comment:
The code mostly looks the same. Is this improving just the styling?

Reply:
yes